src/test/java/httpServer/ListenerAndResponderTest.java

```diff
@@ -22,18 +22,18 @@
     MockServerSocketWrapper mockServerSocket;
     MockBufferedReaderWrapper mockBufferedReader;
     MockPrintWriterWrapper mockPrintWriter;
-    HTTPResponseWriter responseBuilder;
+    HTTPResponseBuilder responseBuilder;
     Map routes;
 
     @BeforeEach
     void setUp() throws IOException {
-        String input = "Accept: */*" + Constants.CRLF +
-                "User-Agent: jUnit" + Constants.CRLF +
-                "Host: localhost:5000" + Constants.CRLF +
-                "GET / HTTP/1.1" + Constants.CRLF + Constants.CRLF;
+        String input = "Accept: */*" + Constants.crlf +
```

Comment:
I think if you do `import static httpServer.http.Constants.crlf` you don't have to qualify the constant each time.